terraform/modules/iam_role_policy/blobstore/policy.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": ["s3:*"],
```

Comment:
![image](https://cloud.githubusercontent.com/assets/604163/24524334/0015d86e-154b-11e7-896f-b26ff660e00b.png)

Can you tighten up this action list to what's actually needed?